### PR TITLE
Maya: Raise PublishValidationError in validators

### PR DESCRIPTION
**Pull request by BigRoy**
## Changelog Description

Improve validation errors to generate an artist-facing report.

## Additional info

The reports are very minor/simple and can be improved over time, but at least now they don't crash the publisher UI.

Also includes the fix of #241 

## Testing notes:

1. Test the relevant validators, they should work.

### CODE REVIEW

File: client/ayon_core/hosts/maya/plugins/publish/validate_vrayproxy.py [hunk 1]
```diff
@@ -17,18 +20,18 @@ def process(self, instance):
         if not self.is_active(data):
             return
         if not data["setMembers"]:
-            raise KnownPublishError(
+            raise PublishValidationError(
```
Comment (iLLiCiTiT) [inline client/ayon_core/hosts/maya/plugins/publish/validate_vrayproxy.py:23]: Should all these be PublishValidationError? User can't do anything about it...

Comment (BigRoy) [reply client/ayon_core/hosts/maya/plugins/publish/validate_vrayproxy.py:23]: I wasn't entirely sure - it felt like something the artists did have control over. 

- If there's no set members, then they just added no objects to the instance and they should add it.
- If frame end is below frame start they should fix it on the instance
- If both vrmesh and alembic are disabled for extraction - that'd also be a toggle for the artist to fix.

Not saying that the validation message is perfect the way they are, but at least I feel like these are something _for the artist_ that they can handle?

Comment (iLLiCiTiT) [reply client/ayon_core/hosts/maya/plugins/publish/validate_vrayproxy.py:23]: I don't know the logic behind, but the message should be modified, at least remove the "This is a bug". If it is a bug it is not artists issue...

Comment (antirotor) [reply client/ayon_core/hosts/maya/plugins/publish/validate_vrayproxy.py:23]: I agree that this is something artist has power to change, but I would remove the "bug" sentence also.

**Review by antirotor — CHANGES_REQUESTED** [review]

**Review by antirotor — APPROVED** [review]
Looks good - it is adding some f-strings but I guess we don't support Maya with python 2.x in AYON anymore.

**Comment by BigRoy** [comment]
> it is adding some f-strings but I guess we don't support Maya with python 2.x in AYON anymore.

Indeed, kill with fire! :)

**Pull request merged**